libyul/optimiser/Metrics.h

```diff
@@ -43,7 +43,6 @@ class CodeSize: public ASTWalker
 public:
 	static size_t codeSize(Statement const& _statement);
 	static size_t codeSize(Expression const& _expression);
-	static size_t codeSize(Block const& _block);
```

Comment:
``Statement`` is implicitly constructible from ``Block``, so IMO this ``codeSize(block)``` should call ``codeSize(Statement{block})`` anyways - however, judging from the VS errors before, Visual C++ might disagree and we need a different fix (e.g. change ``cs.visit(_block);`` to ``cs.visit(Statement{_block});`` above).